gitdb/pack.py

```diff
@@ -410,7 +419,7 @@ def offsets(self):
         if self._version == 2:
             # read stream to array, convert to tuple
             a = array.array('I')    # 4 byte unsigned int, long are 8 byte on 64 bit it appears
-            a.fromstring(buffer(self._cursor.map(), self._pack_offset, self._pack_64_offset - self._pack_offset))
+            _frombytes(a, buffer(self._cursor.map(), self._pack_offset, self._pack_64_offset - self._pack_offset))
```

Comment:
It can be simpler:
```
a = array.array('I', buffer(self._cursor.map(), self._pack_offset, self._pack_64_offset - self._pack_offset))
```
Or maybe even use `memoryview(self._cursor.map())[self._pack_offset : self._pack_64_offset]` instead of `buffer(...)` (not tested).